src/components/Chart/AreaChart.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import { CHART_CONFIG } from './constants';
+
+/**
+ * AreaChart Component for Patternfly React
+ */
+class AreaChart extends C3Chart {
+  static defaultProps = CHART_CONFIG['AreaChart'].defaultConfig;
```

Comment:
CHART_CONFIG.AreaChart.defaultConfig ?

Reply:
I'm surprised Prettier does not format this... I have not seen this style of object access in a while. Redux typically uses upper case style like this `STATUS.PENDING`:
https://github.com/theforeman/foreman/blob/a5a6b5df465560729c86fa4799c1e9ad13941444/webpack/assets/javascripts/react_app/components/statistics/StatisticsChartsList.js#L14

Reply:
This gives better intellisense too ;-)

Reply:
@priley86 👍 

a. using Redux pattern now 

p.s.
I was playing with the Idea of using a variable, this did not work ... and the `[]` access method remained.

do only:
```javascript
class SomeChart extends BaseChart {
  static chart_name = 'SomeChart';
}
```
and in the base:
```javascript
  static defaultProps = CHART_CONFIG[this.chart_name].defaultConfig;

  static get displayName() {
    return this.chart_name;
  }

  get chartConfig() {
    return CHART_CONFIG[this.chart_name];
  }
```